Administrator Guide/Directory Quota.md

```diff
@@ -75,7 +75,12 @@ being treated as "/".
 
     > **Note**
     > In a multi-level directory hierarchy, the strictest disk limit
-    > will be considered for enforcement.
+    > will be considered for enforcement. Also, whenever quota limit
+    > is set for the first time, an auxiliary mount point will be
+    > created under /run/gluster/<VOLNAME>. This is just like any other 
+    > mount point with some special permissions and remains until
+    > quota is disabled, which is being used by quota to set and display
```

Comment:
End the sentence at the word `disabled` and start a new one like this - `This mount point is used by  quota...`